plugin.video.sportschau/default.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import libard
+import resources.lib.jsonparser as jsonparser
+
+class sportschau(libard.libard):

```

Comment:
Using 4 spaces instead of tabs is recommended by pep8 guidelines, probably a good idea to run reindent on your files at some point. Can be done on the next update